autoscheduler/frontend/src/tests/redux/Availability.test.ts

```diff
@@ -473,10 +475,32 @@ describe('Availabilities', () => {
       }];
 
       // act
-      store.dispatch(setAvailabilities(expected));
+      store.dispatch(setAvailabilities(expected, '202031'));
 
       // assert
       expect(store.getState().termData.availability).toEqual(expected);
     });
   });
+
+  describe('skips set availabilities', () => {
+    test('when theres a term mismatch', () => {
```

Comment:
This is really minor but it slightly bugs me how some of the tests with apostrophes are named, here are the 2 best options:
- Use quotes for the string itself (ESlint is fine with this): `"there's"`
- Don't use a contraction: `'there is'`